psi4/src/psi4/libfock/jk.cc

```diff
@@ -618,5 +623,13 @@ void JK::compute() {
         C_right_.clear();
     }
 }
+void JK::set_wcombine(bool wcombine) {
+    wcombine_ = wcombine;
+    if (wcombine) {
+        throw PSIEXCEPTION("To combine exchange terms, use MemDFJK\n");
+    } else {
+        wcombine_ = false;
+    }
+}
```

Comment:
the purpose of this fn is to forbid setting `set_wcombine(true)` unless a derived class overrides it?

can kill off the `else` clause, I think.

Reply:
Yes. That is exactly the purpose.

Reply:
I killed off the else. It's showing up in files changed but not here for whatever reason.